faqs/galaxy/histories_shared.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Finding and working with "Histories shared with me"
+description: How to find and work on histories shared with you
+area: datasets
+layout: faq
+box_type: tip
+contributors: [jennaj, garimavs]
+---
+
+**To find histories shared with me:**
+- Log into your account.
+- Select **User**, in the drop-down menu, select **Histories shared with me**.
+
+**To work with shared histories:**
+- Import the History into your account via copying it to work with it.
+- Unshare Histories that you no longer want shared with you or that you have already made a copy of.
+
+**Note:** Shared Histories (when copied into your account or not) do count in portion toward your total account data quota usage. More details on histories shared concerning account quota usage can be found in [this link](https://galaxyproject.org/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed).
```

Comment:
maybe this link should be also another FAQ?

Reply:
The current link to galaxyproject.org will disappear soon, so the link would be broken. I think that I'd create a new FAQ and once it's merged, link it from here.

Reply:
Okay, is there anything else that I can help with for this PR?

Reply:
I think it's good! Once the other FAQ is added, we can link it from here :)

Reply:
@adi-1999 are you planning to work on [this FAQ](https://galaxyproject.org/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed), we need it to get this other one merged. Thanks!

Reply:
In case @adi-1999 is unable to work on this, then I would like to volunteer for the task.

Reply:
Sure, thanks @garimavs! I've updated [the issue](https://github.com/galaxyproject/galaxy-hub/issues/1262).

Reply:
Hey @beatrizserrano, I have updated the PR. Kindly have a look and suggest any further improvements or amendments.

Reply:
this link is still pointing to the Hub, do we better link the FAQ?

Reply:
Oh, I guess while editing the numbering, I edited the link twice. Sorry for the inconvenience, will undo it.

Reply:
cool, thank you! Ready to merge then! :)